.github/workflows/ci.yml

```diff
@@ -70,19 +70,11 @@ jobs:
         with:
           path: ~\AppData\Local\pip\Cache
           key: ${{ runner.os }}-python-${{ matrix.python-version }}-pip-${{ github.run_id }}-${{ hashFiles('environment.yml') }}
-      - uses: goanpeca/setup-miniconda@v1.6.0
+      - uses: actions/setup-python@v2
         with:
           python-version: ${{matrix.python-version}}
-          channel-priority: strict
-          use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
```

Comment:
we don't need that value anymore?

Reply:
This is used further down the code.

Just to cache pip, we don't need to create a base conda environment. It's slower than using only pip.